packages/gatsby/src/utils/babel-loader-helpers.ts

```diff
@@ -70,17 +94,17 @@ const prepareOptions = (babel, options = {}, resolve = require.resolve) => {
   )
 
   // Go through babel state and create config items for presets/plugins from.
-  const reduxPlugins = []
-  const reduxPresets = []
-  pluginBabelConfig.stages[stage].plugins.forEach(plugin => {
+  const reduxPlugins: PluginItem[] = []
+  const reduxPresets: PluginItem[] = []
+  pluginBabelConfig.stages[stage!].plugins.forEach(plugin => {
     reduxPlugins.push(
       babel.createConfigItem([resolve(plugin.name), plugin.options], {
         name: plugin.name,
```

Comment:
Looks like this is an error. It seems it's correct that `name` isn't a valid key. It accepts just `dirname` and `type`: https://babeljs.io/docs/en/babel-core#createconfigitem

Reply:
Wat do you think would be the best option here? I'm not sure how it will affect prod to remove the `name` key.